code/modules/judaism/rabbi.dm

```diff
@@ -0,0 +1,103 @@
+/*
+PP Station 13 - Judaism Content
+Coded with love by woross
+Sprites hand drawn by KBB
+CONTENTS:
+-Judaism items
+-Rabbi Job
+*/
+
+/area/chapel/synagogue
+	name = "Synagogue"
+
+/obj/item/clothing/head/pp/kippah
+	name = "kippah"
+	desc = "A small Jewish hat."
+	icon_state = "kippah"
+	dynamic_hair_suffix = ""
+
+/obj/structure/sign/poster/contraband/israel
+	name = "Our Greatest Ally"
+	desc = "A poster with the flag of Nanotrasen's greatest ally. It looks quite damaged."
+	icon_state = "poster_israel"
+
+/obj/item/clothing/neck/necklace/dope/rabbi
+	desc = "Feels good to be rich."
+
+/obj/item/clothing/suit/toggle/lawyer/black/rabbi
+	name = "rabbi suit"
+	desc = "A formal suit."
+
+/obj/item/clothing/under/lawyer/black/rabbi
+	name = "formal clothes"
+	desc = "A formal shirt and pants."
+
+/obj/item/clothing/head/pp/jhat
+	name = "Jew hat"
+	desc = "A robust Jewish top hat."
+	icon_state = "jhat"
+
+
+/obj/effect/landmark/start/rabbi
+	name = "Rabbi"
+	icon = 'ppstation/icons/jew.dmi'
+	icon_state = "jewspawn"
+
+/datum/job/rabbi
+	title = "Rabbi"
+	flag = RABBI
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "G-d"
+	selection_color = "#dddddd"
+	access = list(ACCESS_SYNAGOGUE, ACCESS_MAINT_TUNNELS)
```

Comment:
ACCESS_SYNAGOGUE isnt an access

Reply:
Oh, right we'll have to rename the church to the synagogue